src/main/java/br/com/senior/core/base/RequestUtils.java

```diff
@@ -34,12 +33,12 @@
      * @param token AccessToken de authenticação.
      * @throws ServiceException Caso serviço esteja fora ou HTTP Status Code de retorno seja diferente de 2xx.
      */
-    public static void delete(String url, String token) throws ServiceException {
+    public void delete(String url, String token) throws ServiceException {
         try (CloseableHttpClient client = HttpClients.createDefault()) {
             HttpResponse response = executeDelete(url, client, token);
             int statusCode = response.getStatusLine().getStatusCode();
             if (statusCode < 200 || statusCode > 207) {
-                throw new ServiceException(statusCode, String.format("Error to remove Group with HTTP Status code %s", statusCode));
+                throw new ServiceException(statusCode, String.format("Error to remove Entity with HTTP Status code %s", statusCode));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code uses &#39;%s&#39; to format int: statusCode (declared at line 39). This is a potential locale-sensitive handling issue. It might cause errors in the handling and processing of the statement at line: 41. Consider formatting this data with &#39;%d&#39; instead.